paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -431,8 +433,130 @@ class PaymentSheet internal constructor(
          * the network.
          */
         val preferredNetworks: List<CardBrand> = emptyList(),
+
+        internal val allowsRemovalOfLastSavedPaymentMethod: Boolean = true
```

Comment:
I'm honestly not sure why all of these were public to begin with, trying to fix up the future.

Reply:
Should be internal, with `Configuration` offering a `newBuilder()` method.